docs/voltohandson/contenttypesviews.md

```diff
@@ -15,7 +15,7 @@ We will use them as base for further developments on the project.
 
 ## Success Story
 
-Create this content type using `Control Panel`->\`\`Dexterity Content Types\`\`->\`\`Add new content type\`\`.
+Create this content type using `Control Panel` -> `Dexterity Content Types`-> `Add new content type`.
```

Comment:
Although this is an improvement, we should use either of the preferred directives `menuselection` or `guilabel`, depending on whether the item is a menu selection item (navigation) or a GUI item (form inputs and buttons).

I think this one is a menu item.

```suggestion
Create this content type using {menuselection}`Control Panel > Dexterity Content Types > Add new content type`.
```